images/samba-dc/Dockerfile

```diff
@@ -20,14 +20,16 @@ ENV ADMIN_PASSWORD_SECRET=samba-admin-password \
     REALM=ad.example.com \
     SERVER_STRING="Samba Domain Controller" \
     TZ=UTC \
-    WINBIND_TRUSTED_DOMAINS_ONLY=no \
```

Comment:
The samba-dc image had an error message caused by this option. Seems to be removed some years ago:
https://lists.samba.org/archive/samba-technical/2017-December/124265.html

Reply:
Was still just a warning message, thanks for pruning it.